experimental/terraform/setup.sh

```diff
@@ -30,14 +30,22 @@ export BILLING_ACCOUNT=$(gcloud beta billing projects describe ${PARENT_PROJECT}
 echo "Creating new project for Four Keys Dashboard..."
 gcloud projects create ${FOURKEYS_PROJECT} --folder=${PARENT_FOLDER}
 gcloud beta billing projects link ${FOURKEYS_PROJECT} --billing-account=${BILLING_ACCOUNT}
+export PARENT_PROJECTNUM=$(gcloud projects describe ${PARENT_PROJECT} --format='value(projectNumber)')
 
 # FOR DEVELOPMENT ONLY: purge all TF state
+echo "Purging TF state [FOR DEVELOPMENT ONLY]"
 rm -rf .terraform terraform.tfstate* terraform.tfvars
 
+# build the event-handler container (using parent project) and stash it in the fourkeys project
+echo "building event-handler container"
+gcloud services enable cloudbuild.googleapis.com
```

Comment:
do you want to specify `--project=${FOURKEYS_PROJECT}` here too? you're explicit about it in most other commands.

Reply:
Good question! After much tinkering, I found that the most reliable solution was to build the container using the parent project, then push it to the registry in the child project. So it's not necessary to specify the projectID for Cloud Build, but it _is_ necessary for Container Registry. Perhaps I should make it explicit here though to clarify.